make/UNIXUsers.txt

```diff
@@ -0,0 +1,30 @@
+Two options on building UOX3
+
+Opton 1: Automake.sh
+	From this Directory, enter ./automake
+	the uox3 binary will be in this directory upon completion
+	
+Option 2: CMake
+	1) Open a terminal window    
+		- **Windows** users note:  
```

Comment:
Is there value in this note in a file named UNIXUsers.txt?

Reply:
Yes, as when the rest of UOX3 is addressed, it will be useable for ALL OS (so name change at that time)